src/modules/verifiable-credentials/storage-client.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  PresentationDefinition,
+  VerifiableCredential,
+  CredentialSubject,
+} from '@ew-did-registry/credentials-interface';
+import { AxiosInstance } from 'axios';
+import { CacheClient } from '../cacheClient';
+
+export default class VCStorageClient {
+  private _http: AxiosInstance;
+
+  constructor(cacheClient: CacheClient) {
+    this._http = cacheClient.http;
+  }
+
+  async getCredentialsByDefinition(
+    definition: PresentationDefinition
```

Comment:
I think this would be a bit more accurate
```suggestion
  async getCredentialsByPresentationDefinition(
    presentationDefinition: PresentationDefinition
```